compiler/src/emit.rs

```diff
@@ -85,6 +84,20 @@ fn emit_block(
     }
 }
 
+fn emit_assignment(
+    assignment: &Assignment,
+    emitter: &mut Bytecode,
+    cp: &mut ConstantPool,
+    state: &mut EmissionState,
+) {
+    emit(&assignment.rhs, emitter, cp, state);
+    if let Symbol::Local(id) = assignment.identifier {
+        emitter.emit_set_local(id.0 as u8);
+        return;
+    }
+    unimplemented!("cannot support external variables assignations")
```

Comment:
```suggestion
    match assign.identifier {
        Symbol::Local(id) => emitter.emit_set_local(id.0 as u8),
        Symbol::External(_) => {
            unimplemented!("External variable assignations are not implemented yet")
        }
    }
```
Be exhaustive in match patterns.